asserts/fsbackstore.go

```diff
@@ -51,7 +52,7 @@ func OpenFSBackstore(path string) (Backstore, error) {
 }
 
 // guarantees that result assertion is of the expected type (both in the AssertionType and go type sense)
-func (fsbs *filesystemBackstore) readAssertion(assertType *AssertionType, diskPrimaryPath string) (Assertion, error) {
+func (fsbs *filesystemBackstore) readAssertion1(assertType *AssertionType, diskPrimaryPath string) (Assertion, error) {
```

Comment:
I assume the 1 here is a left over from a prior design?  Can probably be moved back now.